docs/components/board/upboard.md

```diff
@@ -67,7 +68,10 @@ Edit and fill in the attributes as applicable.
       "model": "upboard",
       "attributes": {
         "i2cs": ["<your-i2cs-name-1>", "<your-i2cs-name-2>"],
-        "digital_interrupts": ["<your-digital_interrupts-name-1>", "<your-digital_interrupts-name-2>"]
+        "digital_interrupts": [
+          "<your-digital_interrupts-name-1>",
+          "<your-digital_interrupts-name-2>"
+        ]
```

Comment:
Oh! If the array contains 2+ values, multi-line but if single value = single line (like in `docs/components/base/wheeled.md`)?? I like that!

Reply:
I think it's if the values can fit on one line without exceeding 80 chars